README.md

```diff
@@ -25,6 +25,12 @@ In case you want to use a custom app dir name (e.g. `/myapp/`):
 composer create-project --prefer-dist cakephp/app myapp
 ```
 
+If you want to install 4.x branch in your custom app directory
+
+```bash
+composer create-project --prefer-dist --stability=dev cakephp/app:4.x myapp
```

Comment:
Not sure if we should duplicate, instead better modify the existing row maybe?
What do others think?

Reply:
You're right. This isn't master branch. I am directly requesting this pull on 4x branch. It is implied that the user is looking at 4x branch and wants to use 4x. I will make the change.

Reply:
I'm not sure you need to `--stability=dev` 

it works without this , let `cakephp/app` handles this